### DISCUSSION TITLE
Is it safe to delete the older models, such as G_0 and D_0, once the training is finished?

### DISCUSSION THREAD

**Discussion by Mixeykusje** [voicepaw/so-vits-svc-fork#700] [General]
Is it safe to delete the older models, such as G_0 and D_0, once the training is finished? I currently have around four G and D models saved due to a lower eval interval. In my case, I only utilize the latest G model, specifically G_601, and I would like to remove the others to free up storage space.
![speaker](https://github.com/voicepaw/so-vits-svc-fork/assets/131601087/56d6935c-7810-44a7-bb8d-dd9564dbbe3d)

Should I delete the older models only after completing the training, or is it also acceptable to delete them while temporarily suspending the training process? I understand that these models serve as checkpoints, allowing me to resume training from a specific number of epochs onwards. My intention is to optimize hard disk space usage without compromising the model's quality.

**Comment by seoeaa** [comment]
Make a backup and check
